src/ints/mouse.cpp

```diff
@@ -949,8 +949,8 @@ void MOUSE_EventReleased(uint8_t idx)
                                                        get_buttons_joined());
     if (changed_12S) {
         event.request_vmm = MOUSEVMM_NotifyPressedReleased(buttons_12S);
-        event.request_dos = MOUSEDOS_NotifyReleased(buttons_12S, idx_12S, event.id);
-        MOUSESERIAL_NotifyPressedReleased(buttons_12S, idx_12S);
+	event.request_dos = MOUSEDOS_NotifyReleased(buttons_12S, idx_12S);
+	MOUSESERIAL_NotifyPressedReleased(buttons_12S, idx_12S);
     }
```

Comment:
Indentation seems to be off in this file.

Reply:
Fixed.